pages/instructorAssessmentDownloads/instructorAssessmentDownloads.js

```diff
@@ -215,11 +220,15 @@ router.get('/:filename', function (req, res, next) {
     sqldb.query(sql.submissions_for_manual_grading, params, function (err, result) {
       if (ERR(err, next)) return;
       // Replace user-friendly column names with upload-friendly names
-      identityColumn = identityColumn.map((pair) => [pair[1], pair[1]]);
+      identityColumn = (res.locals.assessment.group_work ? groupNameColumn : studentColumn).map(
+        (pair) => [pair[1], pair[1]]
+      );
```

Comment:
What I've done here is I removed some columns that are not useful in the download/upload grading process (name and role). These columns weren't part of the query anyway, so their value was always blank. Given the additional columns added by this PR I took the opportunity to remove them here as well. Given the conflict this would generate with this PR (this change would also imply changes in the docs) I'd prefer to keep this together.

Reply:
Makes sense, thanks for providing the extra context.